src/main/java/chatbot/Ui.java

```diff
@@ -0,0 +1,63 @@
+package chatbot;
+
+import java.util.Scanner;
+
+import chatbot.exceptions.LocalFileException;
+
+/**
+ * Class that represents User Interface which deals with user input and output.
+ */
+public class Ui {
+    private static final String line = "\t____________________________________________________________";
```

Comment:
Since this is used as a constant, would be good if the naming style reflects that: `String LINE`